packages/devtools/web/main.dart

```diff
@@ -5,14 +5,15 @@
 import 'dart:async';
 import 'package:devtools/src/framework/framework_core.dart';
 import 'package:devtools/src/main.dart';
-import 'package:devtools/src/ui/gtags.dart';
+import 'package:devtools/src/ui/analytics.dart' as ga;
 import 'package:platform_detect/platform_detect.dart';
 
 void _gAReportDartExceptions(Exception e, StackTrace stack) {
```

Comment:
just to clarify comment that this exceptions occurring in our app (which are fine to track) not exceptions in the app run by the user which we should be careful about tracking.

Reply:
Right this is only detecting Dart exceptions in DevTools not the user's Flutter app.  I'll add a comment so everyone realizes what's going on here.